src/pages/MyProfile.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { ListGroup, Container } from 'react-bootstrap';
 
-const MyProfile = () => (
-  <div />
-);
+const MyProfile = () => {
+  const { missions } = useSelector((state) => state.missions);
+  const reservedMission = missions.filter((item) => item.reserved === true);
+  console.log(missions);
```

Comment:
Please remove this console.log from the code in order to follow JavaScript best practices, as it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website